### PR TITLE
APR-2809 Implement load live tables from staging

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs [hunk 1]
```diff
@@ -27,8 +30,20 @@ public async Task ClearStagingData()
...
+            await tx.CommitAsync();
+        }
+
+        public async Task LoadDataFromStagingInToLive()
+        {
+            using var tx = await _charitiesDataContext.Database.BeginTransactionAsync();
+
+            await _charitiesDataContext.Database.ExecuteSqlInterpolatedAsync($"TRUNCATE TABLE CharityTrustee");
+            await _charitiesDataContext.Database.ExecuteSqlInterpolatedAsync($"TRUNCATE TABLE Charity");
+            await _charitiesDataContext.Database.ExecuteSqlRawAsync("INSERT INTO Charity SELECT * FROM CharityStaging");
```
Comment (MarkFCain) [inline src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs:45]: If you smoke tested and this worked, no problem with this.  I would expect this not to work cos inserting over the primary key, but if it works, that;s fine.

File: src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs [hunk 2]
```diff
@@ -27,8 +30,20 @@ public async Task ClearStagingData()
...
+            await _charitiesDataContext.Database.ExecuteSqlRawAsync("INSERT INTO CharityTrustee SELECT * FROM CharityTrusteeStaging");
```
Comment (MarkFCain) [inline src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs:47]: same note as above

**Review by MarkFCain — APPROVED** [review]
minor question about smoke testing the inserts. Ignore if smoke testing works

File: src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs:45]: You can insert into primary key as long as it is not an identity column.

File: src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs (hunk 2 above)
Comment (DevenKShah) [reply src/SFA.DAS.Charities.Data/Repositories/CharityImportRepository.cs:47]: You can insert into primary key as long as it is not an identity column.

**Pull request merged**